<ui version="4.0" >
 <class>KonqProfileDlgBase</class>
 <widget class="QWidget" name="KonqProfileDlgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="lblName" >
     <property name="text" >
      <string>&amp;Profile name:</string>
     </property>
     <property name="buddy" >
      <cstring>m_pProfileNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="m_pProfileNameLineEdit" />
   </item>
   <item>
    <widget class="KListWidget" name="m_pListView" />
   </item>
   <item>
    <widget class="QCheckBox" name="m_cbSaveURLs" >
     <property name="text" >
      <string>Save &amp;URLs in profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_cbSaveSize" >
     <property name="text" >
      <string>Save &amp;window size in profile</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
